### PR TITLE
Added system requirements for OVA

### PR DESCRIPTION
**Pull request by stuclem**
Per title. Fixes https://github.com/vmware/vic-product/issues/124.

@anchal-agrawal @mhagen-vmware @frapposelli can you please confirm that these are the only prereqs? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
 
 **Prerequisites**
 
-You downloaded the OVA installer from the [official vSphere Integrated Containers downloads page on vmware.com](http://www.vmware.com/go/download-vic).
+- You downloaded the OVA installer from the [official vSphere Integrated Containers downloads page on vmware.com](http://www.vmware.com/go/download-vic).
+- Deploy the appliance to a vSphere environment that meets the minimum system requirements:
+
+   - 2 vCPUs
+   - 4GB memory
```
Comment (frapposelli) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: 8 gigs of RAM

**Comment by stuclem** [comment]
Thanks @frapposelli. I upped the RAM accordingly. @anchal-agrawal and @mhagen-vmware, anything to add?

**Comment by stuclem** [comment]
Thanks @anchal-agrawal, done!

**Comment by stuclem** [comment]
Going ahead and merging this one

**Pull request merged**